pkg/e2etemplates/aws_operator_chart_values.go

```diff
@@ -21,6 +21,7 @@ const AWSOperatorChartValues = `Installation:
       AWS:
         Region: ${AWS_REGION}
         DeleteLoggingBucket: true
+        IncludeTags: true
```

Comment:
This has an effect of putting our tags to infrastructure components created by e2e tests?

Reply:
In fact only the s3 bucket ones, we added recently the functionality to exclude them (they are not allowed in China regions). By default installations set this flag to true, this has the same effect.